### DISCUSSION TITLE
HTML overlay: possible to scale text with images?

### DISCUSSION THREAD

**Discussion by DevinSmithWork** [openseadragon/openseadragon#2288] [General]
I'm wondering if there's a way to scale the text size in HTML overlays dynamically as the user zooms an image in and out?

Thanks!

**Comment by msalsbery** [comment]
> I'm wondering if there's a way to scale the text size in HTML overlays dynamically as the user zooms an image in and out?
> 
> 
> 
> Thanks!

You could subscribe to the [Viewer zoom event](https://openseadragon.github.io/docs/OpenSeadragon.Viewer.html#.event:zoom) and adjust the HTML overlay element style

**Comment by iangilman** [comment]
Another option would be to use one of the other overlay [plugins](https://openseadragon.github.io/#plugins), like the [SVG](https://github.com/openseadragon/svg-overlay) or [HTML](https://github.com/openseadragon/html-overlay) ones. They scale all of their content as you zoom.
